src/index.html

```diff
@@ -10,7 +10,44 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="form"
+      data-qa="big"
+    >
+
+      <input
+        type="text"
+        name="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="
+          form__input
+          form__input--big
```

Comment:
it is redundant modifier. set all css properties for form__input element and reset neccassary for form__input--small modifier
```suggestion
```